optuna/importance/_fanova/_evaluator.py

```diff
@@ -0,0 +1,123 @@
+    .. note::
+
+        Requires the `sklearn <https://github.com/scikit-learn/scikit-learn>`_ Python package.
+
+    .. note::
+
+        Pairwise and higher order importances are not supported through this class. They can be
+        computed using :class:`~optuna.importance._fanova._fanova._Fanova` directly but is not
+        recommended as interfaces may change without prior notice.
+
+    Args:
+        n_estimators:
+            The number of trees in the forest.
+        max_depth:
+            The maximum depth of the trees in the forest.
+        random_state:
+            Controls the randomness of the forest. For deterministic behavior, specify a value
+            other than :obj:`None`.
+
+    """
+
+    def __init__(
+        self, n_estimators: int = 16, max_depth: int = 64, random_state: Optional[int] = None
+    ) -> None:
+        self._evaluator = _Fanova(
+            n_estimators=n_estimators,
+            max_depth=max_depth,
+            min_samples_split=2,
+            min_samples_leaf=1,
+            random_state=random_state,
+        )
+
+    def evaluate(self, study: Study, params: Optional[List[str]]) -> Dict[str, float]:
+        distributions = _get_distributions(study, params)
+        params_data, values_data = _get_study_data(study, distributions)
+
+        if params_data.size == 0:  # `params` were given but as an empty list.
+            return OrderedDict()
+
+        search_spaces = numpy.empty((len(distributions), 2), dtype=numpy.float64)
+        search_spaces_is_categorical = []
+
+        for i, distribution in enumerate(distributions.values()):
+            if isinstance(distribution, CategoricalDistribution):
+                search_spaces[i, 0] = 0
+                search_spaces[i, 1] = float(len(distribution.choices))
+                search_spaces_is_categorical.append(True)
+            elif isinstance(
+                distribution,
+                (
+                    DiscreteUniformDistribution,
+                    IntUniformDistribution,
+                    LogUniformDistribution,
+                    UniformDistribution,
+                ),
+            ):
+                search_spaces[i, 0] = distribution.low
+                search_spaces[i, 1] = distribution.high
+                search_spaces_is_categorical.append(False)
+            else:
+                assert False
+
+        evaluator = self._evaluator
+        evaluator.fit(
+            X=params_data,
+            y=values_data,
+            search_spaces=search_spaces,
+            search_spaces_is_categorical=search_spaces_is_categorical,
+        )
+
+        individual_importances = {}
+        for i, name in enumerate(distributions.keys()):
+            importance = evaluator.quantify_importance((i,))
+            individual_importance = importance[(i,)]["individual importance"]
+            individual_importances[name] = individual_importance
+
+        tot_importance = sum(v for v in individual_importances.values())
```

Comment:
```suggestion
        total_importance = sum(v for v in individual_importances.values())
```
How about renaming the variable like this for clarity?

Reply:
I agree that "total" is better. Thanks!